datasets/mx/governors/mx_governors.yaml

```diff
@@ -2,8 +2,11 @@ title: Mexico Governors
 entry_point: crawler.py
 prefix: mx-gov
 coverage:
-  frequency: weekly
+  frequency: never
   start: "2024-02-03"
+  end: "2029-01-01" # After every term has ended, to avoid warnings because of occupancy end dates
```

Comment:
instead, set the real end date of coverage, set no_end_implies_current, and don't set end_date because we have no reason to believe these are currently in the post.

See a similar commit here 4e1ae63b407ca2411720e344555e83863ac5d387

Reply:
exactly. If we know a list is maintained, we can assume that people on the list are currently members. If it's not being maintained, we know it's going to be incorrect at some point to call them current, and we don't know when. That's why that warning is connected to the "assume" argument.

the dataset coverage date tells us when updates ended so we don't want to set that into the future.